pii-filtering/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'PII Filtering'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+[Personally Identifiable
+Information](https://csrc.nist.gov/glossary/term/personally_identifiable_information)
+(PII) about your users must be handled very carefully in order to respect your
+users' privacy and to comply with laws and regulations your organization is
+subject to. When a third-party tool like pganalyze processes your database
+statistics and logs, it may involve handling data that includes this PII.
+
+In order to strike the right balance between what you share with pganalyze and
+the impact on monitoring, statistics, and insights available through our
+service, we have several collector settings that control metadata collection.
+
+Note that this filtering happens in the collector, the open-source agent that
+connects to your database and runs on your infrastructure. Filtering occurs
+before the data ever reaches pganalyze.
+
+
+## Filtering settings
+
+The `filter_log_secret` setting determines how content coming from logs is filtered:
```

Comment:
I wonder if it'd be nice to cross reference this to the above collector setting page. e.g. saying "The `filter_log_secret` setting [in the collector](link to the setting section) determines..."

Reply:
We can't link to individual settings, but you're right, we should link back to that settings section somewhere.